Makefile

```diff
@@ -317,6 +322,8 @@ release.sidecar:
 	@cp $(BINDIR)/$(SIDECAR_APP_NAME) $(BUILDDIR)/usr/bin/
 	@cp openresty/*.tar.gz $(BUILDDIR)/opt/openresty_dist/
 	@tar -C $(BUILDDIR) -czf $(RELEASEDIR)/$(SIDECAR_RELEASE_NAME).tar.gz --transform 's:^./::' .
+	# Make a copy for examples testing
+	@cp $(RELEASEDIR)/$(SIDECAR_RELEASE_NAME).tar.gz $(RELEASEDIR)/a8sidecar-current.tar.gz
```

Comment:
This will end up in the github public release. You can reuse the existing files in the release dir right ?